slsim/Sources/supernovae_host_match.py

```diff
@@ -33,35 +33,11 @@ def match(self):
         :return: catalog with supernovae redshifts and their corresponding host galaxies
         :return type: astropy Table
         """
+        dtype_tuple = tuple(["float64"] * len(self.galaxy_catalog.colnames))
```

Comment:
it's hard to read what's happening here in the code. Can you document the code here?